CRM/Admin/Form/ContactType.php

```diff
@@ -147,6 +147,7 @@ public function postProcess() {
     }
 
     $contactType = CRM_Contact_BAO_ContactType::writeRecord($params);
+    CRM_Utils_System::flushCache();
```

Comment:
If actually needed, then this belongs in `\CRM_Contact_BAO_ContactType::self_hook_civicrm_post`, as this form bypasses the api.

Although that function already has this:
https://github.com/civicrm/civicrm-core/blob/ea8ae7ad83fb6ccf68a7f1f4cd0a56464bfb78e7/CRM/Contact/BAO/ContactType.php#L486

Reply:
Thanks - removed the flush() from here. I confirmed by enabling/disabling and then going to Find Contacts to make sure that things seemed OK.